openreview/venue_request/venue_request.py

```diff
@@ -964,6 +969,11 @@ def setup_registration_stages(self):
                 'required': False,
                 'order': 12
             },
+            'ac_registration_name': {
```

Comment:
should we call it "area_chair_registration_name" or "AC_regristation_name"?  so the UI shows them as "Area Chair Registration Name" or "AC Registration Nname"